requirements.txt

```diff
@@ -9,7 +9,7 @@ requests>=2.20.0
 python-Levenshtein>=0.12.0
 pytz>=2018.9
 whatthepatch>=0.0.5
-icalevents==0.1.25
+icalevents==0.1.24
```

Comment:
What is the bug in 0.1.25?

Reply:
Some problems when comparing datetime with and without timezone.

Reply:
Sun Apr 15 1707 03:14:15

Reply:
ahah